src/data-display/tags/story-helper.ts

```diff
@@ -91,16 +136,52 @@ export const getTagArgTypes = (): ArgTypes => ({
             type: 'text',
         },
     },
-    //
-    default: {
+    /* slots */
+    defaultSlot: {
         name: 'default',
-        description: 'Slot for text.',
-        defaultValue: 'tag name',
+        description: 'Slot for replace contents composed of key, value, category',
+        defaultValue: null,
         table: {
             type: {
                 summary: null,
             },
-            defaultValue: {
+            category: 'slots',
+        },
+    },
+    categorySlot: {
+        name: 'category',
+        description: 'Slot for replace category item',
```

Comment:
```suggestion
        description: 'Slot to replace category item',
```

I think it must be better :)